[2023-08-24T03:54:22.839+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: nikeProducts_dag.step_2_task manual__2023-08-24T03:54:19.561254+00:00 [queued]>
[2023-08-24T03:54:22.847+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: nikeProducts_dag.step_2_task manual__2023-08-24T03:54:19.561254+00:00 [queued]>
[2023-08-24T03:54:22.847+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-08-24T03:54:22.848+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 2
[2023-08-24T03:54:22.848+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-08-24T03:54:22.858+0000] {taskinstance.py:1383} INFO - Executing <Task(PythonOperator): step_2_task> on 2023-08-24 03:54:19.561254+00:00
[2023-08-24T03:54:22.864+0000] {standard_task_runner.py:55} INFO - Started process 242 to run task
[2023-08-24T03:54:22.867+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'nikeProducts_dag', 'step_2_task', 'manual__2023-08-24T03:54:19.561254+00:00', '--job-id', '48', '--raw', '--subdir', 'DAGS_FOLDER/nikeProducts_dag.py', '--cfg-path', '/tmp/tmpkcybx96l']
[2023-08-24T03:54:22.870+0000] {standard_task_runner.py:83} INFO - Job 48: Subtask step_2_task
[2023-08-24T03:54:22.920+0000] {task_command.py:376} INFO - Running <TaskInstance: nikeProducts_dag.step_2_task manual__2023-08-24T03:54:19.561254+00:00 [running]> on host 5879ed69cda2
[2023-08-24T03:54:22.971+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=nikeProducts_dag
AIRFLOW_CTX_TASK_ID=step_2_task
AIRFLOW_CTX_EXECUTION_DATE=2023-08-24T03:54:19.561254+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-24T03:54:19.561254+00:00
[2023-08-24T03:54:22.982+0000] {logging_mixin.py:120} INFO - products_etl.csv
[2023-08-24T03:54:23.056+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 175, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/operators/python.py", line 193, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/scripts/transform_data.py", line 19, in main
    df['rating'] = df['rating'].astype(int)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/generic.py", line 5815, in astype
    new_data = self._mgr.astype(dtype=dtype, copy=copy, errors=errors)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/internals/managers.py", line 418, in astype
    return self.apply("astype", dtype=dtype, copy=copy, errors=errors)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/internals/managers.py", line 327, in apply
    applied = getattr(b, f)(**kwargs)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/internals/blocks.py", line 591, in astype
    new_values = astype_array_safe(values, dtype, copy=copy, errors=errors)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/dtypes/cast.py", line 1309, in astype_array_safe
    new_values = astype_array(values, dtype, copy=copy)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/dtypes/cast.py", line 1257, in astype_array
    values = astype_nansafe(values, dtype, copy=copy)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/dtypes/cast.py", line 1168, in astype_nansafe
    return astype_float_to_int_nansafe(arr, dtype, copy)
  File "/home/airflow/.local/lib/python3.7/site-packages/pandas/core/dtypes/cast.py", line 1214, in astype_float_to_int_nansafe
    "Cannot convert non-finite values (NA or inf) to integer"
pandas.errors.IntCastingNaNError: Cannot convert non-finite values (NA or inf) to integer
[2023-08-24T03:54:23.067+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=nikeProducts_dag, task_id=step_2_task, execution_date=20230824T035419, start_date=20230824T035422, end_date=20230824T035423
[2023-08-24T03:54:23.075+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 48 for task step_2_task (Cannot convert non-finite values (NA or inf) to integer; 242)
[2023-08-24T03:54:23.120+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-08-24T03:54:23.148+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
